### ISSUE TITLE
Implicit tile height not taken into account in ImageService3 description

### ISSUE THREAD

**Issue by LvanWissen** [allmaps/allmaps#157]
**Describe the bug**
A minimal ImageService3 info.json description is allowed to only contain a `width` key in a `tiles` object. If the `height` key is left out, it means that it is the same as the width. See: https://iiif.io/api/image/3.0/#54-tiles

The Allmaps Editor is not able to process this, which leads to a warped/stretched image (see screenshot).

**To reproduce**
A minimal `info.json` example:

```json
{
  "@context": "http://iiif.io/api/image/3/context.json",
  "id": "https://iiif.micr.io/vlYLs",
  "type": "ImageService3",
  "tiles": [
    {
      "scaleFactors": [
        1,
        2,
        4
      ],
      "width": 1024
    }
  ],
  "profile": "level2",
  "protocol": "http://iiif.io/api/image",
  "height": 5468,
  "width": 6294
}
```

A working example with the `height` key:

```json
{
  "@context": "http://iiif.io/api/image/3/context.json",
  "id": "https://iiif.micr.io/vlYLs",
  "type": "ImageService3",
  "tiles": [
    {
      "scaleFactors": [
        1,
        2,
        4
      ],
      "width": 1024,
      "height": 1024
    }
  ],
  "profile": "level2",
  "protocol": "http://iiif.io/api/image",
  "height": 5468,
  "width": 6294
}

```

**Screenshots**
![image](https://github.com/user-attachments/assets/41138f4b-f7f0-488a-803e-0ba3976630cb)

**Device information:**
Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:132.0) Gecko/20100101 Firefox/132.0

**Comment by bertspaan** [comment]
Can't reproduce this with the latest version of `@allmaps/iiif-parser`. Copying each of your examples above and then running `pbpaste | allmaps iiif parse` yields the same results. Probably Allmaps Editor uses an older version of the IIIF parser.

**Comment by LvanWissen** [comment]
Indeed, the image is correctly shown when loaded in https://iiif.allmaps.org/.

**Comment by sammeltassen** [comment]
Closing as fixed in latest versions (might still happen in Viewer, but a new version of Viewer will be published soon)
